docs/index.rst

```diff
@@ -23,6 +23,24 @@ sunkit-image
           generated/gallery/index
           changelog
 
+Mission Statement
+=================
+
+The goal of the ``sunkit-image`` library is to provide access to image processing routines that are:
+
+1. Focused on being applied to solar physics data.
+2. Are published in the literature or in preparation to be published.
+3. If there is no publication, it must be widely used throughout the solar physics community.
+   One example might be co-alignment routines that exist in SSWIDL.
+
+As this is a SunPy affiliated package, the inputs to these routines are required to be Maps (sunpy) or ANDCubes (ndcube).
+
+The ownership of the code is also important.
+If the code is already in a released package, we will wrap it in a way that makes it easy to use with Maps or ANDCubes but otherwise leave it alone.
+We will not copy code from other packages into this one.
+
+Finally, this library is not aiming to be a general purpose image processing library.
```

Comment:
Could we move this to the beginning? Or maybe have a subsection on what sunkit-image is not?

Reply:
Yeah I can move this to the start.